resume-input/src/Components/Basics/Basics.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './basics.scss';
+
+
+export default function Basics(props) {
+
+

```

Comment:
too much white space, people will have to scroll more
I used to get dinged for this :)

Reply:
Haha normally I wouldn't have much space but I was out of time. Thanks for pointing that out though!